src/Resolver/InjectionInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-di for the canonical source repository
+ * @copyright Copyright (c) 2018 Zend Technologies USA Inc. (https://www.zend.com)
+ * @license   https://github.com/zendframework/zend-di/blob/master/LICENSE.md New BSD License
+ */
+
+namespace Zend\Di\Resolver;
+
+use Psr\Container\ContainerInterface;
+
+/**
+ * Defines the injection to perform for a parameter
+ */
+interface InjectionInterface
+{
+    /**
+     * @return mixed The resulting injection value
+     */
+    public function toValue(ContainerInterface $container);
+
+    public function export() : string;
+
+    public function isExportable() : bool;
```

Comment:
Document the purpose behind the above two methods. What does `export()` do? Can injections be non-exportable? what should `export()` do in such cases? etc.

Reply:
Not sure why github left this open, but this was addressed - see changeset